Troupon.Catalog.Api/Startup.cs

```diff
@@ -93,8 +93,8 @@ public void Configure(IApplicationBuilder app, IServiceProvider serviceProvider,
       // app.UsePathBase("/graphql");
 
       // catalogDbContext.Database.EnsureDeleted();
-      var catalogDbContext = dbContextFactory.CreateDbContext();
-      catalogDbContext.Database.Migrate();
+      // var catalogDbContext = dbContextFactory.CreateDbContext();
```

Comment:
Please remove commented out code. If we need to bring it back, it's in the git repository.

Reply:
Done!